A14-channelz.md

```diff
@@ -701,16 +701,18 @@ message Subchannel {
   repeated SocketRef socket = 5;
 }
 
```

Comment:
Consider adding a simple doc comment pointing to the connectivity state API page here: https://github.com/grpc/grpc/blob/master/doc/connectivity-semantics-and-api.md

Reply:
Done